forge-gui/res/cardsfolder/upcoming/golden_argosy.txt

```diff
@@ -0,0 +1,15 @@
+Name:Golden Argosy
+ManaCost:4
+Types:Artifact Vehicle
+PT:3/6
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberAllPumped$ True
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigFlicker | TriggerDescription$ Whenever CARDNAME attacks, exile each creature that crewed it this turn. Return them to the battlefield tapped under their owner’s control at the beginning of the next end step.
+SVar:TrigFlicker:DB$ ChangeZoneAll | ChangeType$ Creature.IsRemembered | Mandatory$ True | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig
```

Comment:
there should be a "Crewed this Turn" Property, so you don't need RemeberCrew

Reply:
Sorry, are you able to elaborate more? I couldn't find a `CrewedThisTurn` property across current card scripts. Is this what you say you are fixing?